pubspec.yaml

```diff
@@ -139,6 +143,9 @@ dependencies:
 
   weekday_selector: ^1.1.0
 
+  # 相機
```

Comment:
Suggest DON't use Chinese for comment, use English instead. The previous comments for other existing package does not plays a good example, don't follow them.
And also suggest not just repeat the package name since it is just a redundant comment without any meaningful descriptions.

Reply:
resolve commit : [11f249f](https://github.com/NEO-TAT/tat_flutter/pull/83/commits/11f249fe04736ad45ba7c7b20e48773baa558764)